<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyCreateActivity</class>
 <widget class="QWidget" name="MyCreateActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>活动名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>282</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_update">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>修改活动</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_abstract">
        <property name="text">
         <string>活动摘要</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>包含子活动</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="MyListWidget_child" name="listWidget_child">
        <property name="spacing">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyListWidget_child</class>
   <extends>QListWidget</extends>
   <header location="global">mylistwidget_child.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
